static/js/narrow.js

```diff
@@ -658,6 +663,8 @@ exports.narrowed_by_reply = function () {
     var operators = current_filter.operators();
     return ((operators.length === 1 &&
              current_filter.operands("pm-with").length === 1) ||
+            (operators.length === 1 &&
+             current_filter.operands("huddle-with").length === 1) ||
```

Comment:
I think this shouldn't be here -- the point of `narrowed_by_reply` is to catch cases where the narrow completely specifies where a reply would go, and that isn't true for `huddle-with`.